_pages/projects.md

```diff
@@ -47,7 +47,7 @@ Template:
 * Team 10 Project
 
 ![team-11.png](/assets/images/projects/team-11.png){:class="img-final-projects"}
-* Team 11 Project
+* [**GPU-VPM**](https://github.com/DomIno0o/CIS5650-Final-Project-GPU-VPM) -- [Dominik Kau](https://www.linkedin.com/in/dominikkau/ ), [Nadine Adnane](https://www.linkedin.com/in/nadnane/), [Shreyas Singh](https://linkedin.com/in/shreyassinghiitr)
```

Comment:
Can you expand "VPM" or add a brief description? Its may not necessaily be a term most people are familiar with.

While I can merge the image you have submitted, we want to have something that you have generated. Please be sure to update this when you have one ready.